caravel/assets/javascripts/explorev2/index.jsx

```diff
@@ -2,12 +2,31 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import ExploreViewContainer from './components/ExploreViewContainer';
 
+import { compose, createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import { initialState } from './stores/store';
+
 const exploreViewContainer = document.getElementById('js-explore-view-container');
 const bootstrapData = exploreViewContainer.getAttribute('data-bootstrap');
 
+
+import { exploreReducer } from './reducers/exploreReducer';
+import persistState from 'redux-localstorage';
+
+let enhancer = compose(persistState());
+if (process.env.NODE_ENV === 'dev') {
+  enhancer = compose(
+    persistState(), window.devToolsExtension && window.devToolsExtension()
+  );
+}
+let store = createStore(exploreReducer, initialState, enhancer);
+
 ReactDOM.render(
-  <ExploreViewContainer
-    data={bootstrapData}
-  />,
+  <Provider store={store}>
+    <ExploreViewContainer
+      data={bootstrapData}
```

Comment:
is there any data from the server in `bootstrapData` that can be used to bootstrap the initial state? i think we get `vizType` and `datasourceID`